modules.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "nf-core/epitopeprediction",
+    "homePage": "https://github.com/nf-core/epitopeprediction",
+    "repos": {
+        "nf-core/modules": {
+            "custom/dumpsoftwareversions": {
+                "git_sha": "20d8250d9f39ddb05dfb437603aaf99b5c0b2b41"
+            },
+            "fastqc": {
```

Comment:
We could also remove the fastqc related files, right?

Reply:
done in a separate PR